returnn/tf/layers/basic.py

```diff
@@ -4565,10 +4569,10 @@ def __init__(self, kind, sources, activation=None, with_bias=False,
                eval=None, eval_locals=None, eval_for_output_loss=False,
                **kwargs):
     """
-    :param str kind: e.g. "average" or "add", or "eval"
+    :param str kind: accepted values are `average`, `add`, `sub`, `mul`, or `eval`
```

Comment:
There are much more accepted values. That's why I wrote "e.g.".

Reply:
Ah, sorry, not in current implementation, I think. But this sounds again like it would be restricted to that. But sth like `kind="div"` should also be possible, it's just not implemented yet.

Reply:
Actually, you set an assert for those values `assert kind in ["average", "add", "sub", "mul", "eval"]`

Reply:
Yea because it's not implemented otherwise. Maybe I should have raised `NotImplementedError` instead. I was just too lazy to type that.